Am I on Heroku? nah
lol undefined
Node app is running on port 3000
!! ERROR: connect ECONNREFUSED 127.0.0.1:3306
Stack: SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:3306
    at Handshake._callback (/Users/elliotfiske/commitment/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:79:20)
    at Handshake.Sequence.end (/Users/elliotfiske/commitment/node_modules/mysql/lib/protocol/sequences/Sequence.js:85:24)
    at Protocol.handleNetworkError (/Users/elliotfiske/commitment/node_modules/mysql/lib/protocol/Protocol.js:364:14)
    at Connection._handleNetworkError (/Users/elliotfiske/commitment/node_modules/mysql/lib/Connection.js:434:18)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at emitErrorNT (net.js:1253:8)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickDomainCallback (node.js:397:17)
EXTREMELY UNLIKELY ERROR DETECTED "connect ECONNREFUSED 127.0.0.1:3306" SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:3306
    at Handshake._callback (/Users/elliotfiske/commitment/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:79:20)
    at Handshake.Sequence.end (/Users/elliotfiske/commitment/node_modules/mysql/lib/protocol/sequences/Sequence.js:85:24)
    at Protocol.handleNetworkError (/Users/elliotfiske/commitment/node_modules/mysql/lib/protocol/Protocol.js:364:14)
    at Connection._handleNetworkError (/Users/elliotfiske/commitment/node_modules/mysql/lib/Connection.js:434:18)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at emitErrorNT (net.js:1253:8)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickDomainCallback (node.js:397:17)
